govcd/api_vcd_test.go

```diff
@@ -293,7 +294,7 @@ func (vcd *TestVCD) SetUpSuite(check *C) {
 		skipVappCreation = true
 	}
 	// creates a new VApp for vapp tests
-	if !skipVappCreation && config.VCD.Network != "" && config.VCD.StorageProfile.SP1 != "" &&
+	if !skipVappCreation && config.VCD.Networks[0] != "" && config.VCD.StorageProfile.SP1 != "" &&
```

Comment:
Before using `config.VCD.Networks[0]` we need to check that `config.VCD.Networks` has been initialised

Reply:
Also, this is a good place to check whether the network names are different (i.e. user did not provide the same name twice)